pkgs/tools/admin/awscli/default.nix

```diff
@@ -7,6 +7,13 @@
 let
   py = python3.override {
     packageOverrides = self: super: {
+      colorama = super.colorama.overridePythonAttrs (oldAttrs: rec {
+        version = "0.3.9";
```

Comment:
0.4.1 would also be ok

Reply:
Hmm, interesting, seems to work.

I was looking at the wrong place:
https://github.com/aws/aws-cli/blob/1.16.266/setup.py#L46
Instead of:
But I find it strange due to: https://github.com/aws/aws-cli/blob/1.16.266/setup.py#L49

Will update to this, that explains why this didn't break until just recently, was a bit surprised as how to it was working up until now.